docs/01-installing.md

```diff
@@ -3,7 +3,12 @@
 ## Prerequisites
 
 - Make sure you have installed the latest version of [`Neovim 0.5`](https://github.com/neovim/neovim/releases/tag/v0.5.0).
-- Have `npm`, `node` and `cargo` installed on your system.
+- Have these programs installed on your system and accessible through the terminal under their name:
+  - `npm`
+  - `node`
+  - `cargo`
+  - `xclip`
```

Comment:
```suggestion
  - `rg`
```

Reply:
I don't quite understand where the connection between `cargo` and `xclip` is. When I installed LunarVim with `cargo` as a dependency, `xclip` was still missing.